### DISCUSSION TITLE
Add modified-date of blogpost to pdf

### DISCUSSION THREAD

**Discussion by ghost** [quarto-dev/quarto-cli#5570] [Q&A]
In the folder of my posts, in the _metadata.yml I have `date-modified: last-modified`

This makes, in the html output to appear the date I set in the post + the last-modified date.

I want that in my pdf output but I only obtain the date but not date + last-modifed date. Anyway to do this? I have added this in the _metadata.yml  without success: 
format:
  html: default 
  pdf:
    date-modified: last-modified
    citation: true
```

**Comment by mcanouil** [comment]
`date-modified` is only for HTML as it is not part of the PDF template.
For this to work, you need to modify the partials, see <https://quarto.org/docs/journals/templates.html#latex-partials>, especially, the `title.tex` partial (https://github.com/quarto-dev/quarto-cli/blob/main/src/resources/formats/pdf/pandoc/title.tex).

---

Please be careful of the formatting of your post.
Also we highly encourage to provide a working reproducible example, _a.k.a._, a complete Quarto document, which allows to avoid making assumption on what might be your document, options, _etc._.
Additionally, it allows community member to spend more time answering the question rather than spending time on trying to create an example from scratch.

**Reply by mcanouil** [reply]
Did you try to modify the partial as suggested?

Also you might be interested in linking your PDF to
Your blog post, see <https://quarto.org/docs/output-formats/html-multi-format.html>.

**Reply by ghost** [reply]
I did not try modifying the partial because I do not have any idea of how that works. Could you reference any good documentation for learning about this?

**Reply by mcanouil** [reply]
The links were provided earlier, please take a look at the previous comments.
